### ISSUE TITLE
[14.0, 13.0] [BUG] Changing the Width of One Column on a Tree View Sometimes Changes the Width of All Columns

### ISSUE THREAD

**Issue by slsanc** [odoo/odoo#72197]
Impacted versions:
13.0, 14.0

Steps to reproduce:

1. Go to https://demo.odoo.com/ or a standard Odoo environment
2. go to the Manufacturing App
3. Go to the Manufacturing Orders Tree View
4. Click the Optional Columns Drop-down and Select all of the optional columns, so that the table overflows off of the right-hand side of the screen, and a scroll-bar shows up at the bottom
5. Attempt to re-size the width of the "Product" column by Clicking and Dragging on the right-hand border of that column's header.

Current behavior:

 * The width of every column is changed

Expected behavior:

 * Only the width of the "Product" column is changed

**Comment by pedrobaeza** [comment]
This is something I have faced as well, but with not proper steps to reproduce by my part, but it seems you have them, so tagged for the Odoo team.

**Comment by MaximeDelarue** [comment]
I have found the behavior you are describing only happens on **Firefox**.
On **Chrome** only the width of the targeted column is changed, as expected.

Is it the case on your end too?

**Comment by koulchi** [comment]
I have found the behavior you are describing only happens on Firefox.
On Chrome only the width of the targeted column is changed, as expected.

Is it the case on your end too?

**Comment by pedrobaeza** [comment]
Yes, I use Firefox, but it happens sometimes. I reproduce it consistently though with the steps given in the main comment.

**Comment by koulchi** [comment]
However, with Chrome we can't reproduce the described behavior

**Comment by pedrobaeza** [comment]
Then it's a Firefox specific thing to be fixed for that browser.

**Comment by mkxmawilix** [comment]
Hello there,

I have the same problem on chrome for a Customer, I think it appear when you have a large description.
I can reproduce it here : https://demo3.odoo.com/web#id=48&action=901&model=sale.order&view_type=form&cids=1&menu_id=632

Just copy/past many time the description to enlarge the size, save, reload the page and try to resize the description field and all columns would be resized too.

I hope that can help :)

**Comment by koulchi** [comment]
Mmmh my bad, I haven't tested with the new chrome version. 

 I confirm, since new Chrome version, this bug also happens ..

**Comment by mkxmawilix** [comment]
There is a workaround at the moment, if we click two times on the column name for instance to sort the values the problem disappear until the next page reloading :)

**Comment by traviswaelbro** [comment]
The workaround doesn't appear to work for me in either Firefox 89.0 or Chrome 91.0.4472.106 on Mac.

**Comment by ekaleonardo619** [comment]
Hi, can you please try on stock.move.line model (Inventory ->  Reporting -> Product Moves)? 
This case keeps happened on that tree view

**Comment by msh-odoo** [comment]
@slsanc @pedrobaeza I guess it should be fixed with #73636
